src/core/dfn.js

```diff
@@ -26,9 +25,36 @@ export function run() {
     }
 
     // Only add `lt`s that are different from the text content
-    if (titles.length === 1 && titles[0] === norm(dfn.textContent)) {
+    if (titles.length === 1 && firstTitle === norm(dfn.textContent)) {
       return;
     }
     dfn.dataset.lt = titles.join("|");
   });
 }
+/**
+ *
+ * @param {string} title
+ * @param {HTMLElement} dfn
+ */
+function processInternalSlotsAsIDL(title, dfn) {
```

Comment:
Although webref gives IDL attribute and internal slots the same `type: "attribute"`, an internal slot is not an IDL attribute. So I would rename this to something like `associateInternalSlotWithIDL`.

Reply:
see also my remark at https://github.com/w3c/respec/issues/3644#issuecomment-864894696